client/js/components/statement/assessmentTable/DpFragmentsSwitcher.vue

```diff
@@ -157,6 +160,10 @@ export default {
   },
 
   methods: {
+    hasFragmentFilters (filters) {
+      return filters.length ? filters.some((filter) => filter.type !== 'statement') : false
```

Comment:
is there more than one type "fragment"? I not, I would prefere `filter.type === 'fragment'` (or what ever the name of the fragment-Ressource is)
```suggestion
      return filters.length ? filters.some(filter => filter.type !== 'statement') : false
```

Reply:
10ef30c874444adbde64b7479049fa53b2e44cbc